README.adoc

```diff
@@ -48,28 +48,27 @@ public interface ProductRepository extends PageableRepository<Product, Long> {
 }
 ----
 
-
 === Maven
 
 To start to use Jakarta Dara, add the API as a Maven dependency:
 
 [source,xml]
 ----
 <dependency>
-  <artifactId>jakarta-data-api</artifactId>
-  <groupId>jakarta.data</groupId>
-  <version>1.0.0-SNAPSHOT</version>
+    <groupId>jakarta.data</groupId>
+    <artifactId>jakarta-data-api</artifactId>
+    <version>1.0.0-b2</version>
```

Comment:
Everything else in this pull looks good, but is it correct to hard code the version here?  Will this confuse the automation (once added) for building new releases?

Reply:
It is a good point; I saw the JMolecules readme and replicated it here:

https://github.com/xmolecules/jmolecules

It keeps the latest version, so we must update it manually every time.


Another option would be put `${version}`  as [Spring did](https://github.com/spring-projects/spring-data-mongodb#maven-configuration).


Even with more work, I like keeping the latest version approach.

Please, let me know what your thoughts.

Reply:
It's fine if you'd like to hard code it for now.  If, when working on the build automation, you need to switch it back, that can easily be done.